Ginger/Ginger/ApplicationModelsLib/POMModels/POMEditPage.xaml.cs

```diff
@@ -220,9 +220,8 @@ private void FillTargetAppsComboBox()
                     Reporter.ToUser(eUserMsgKey.MissingTargetApplication, "The mapped " + mPOM.Key.ItemName + " Target Application was not found, please select new Target Application");
                 }
             }
+            xTargetApplicationComboBox.ItemsSource = null;
```

Comment:
Same code exists on the start of SupportedTargetApplication() functoin, do we need it here?

Reply:
yes, earlier it was in that method but when we add the new TA at that time existing TA list Gets empty due to that we faced the issue.

Reply:
so i have added here